db/migrate/20150103105333_tag_aato_cleanup.rb

```diff
@@ -0,0 +1,13 @@
+class TagAatoCleanup < ActiveRecord::Migration
+  def up
+   ActsAsTaggableOn::Tag.delete_all(taggings_count: 0)
+   ActsAsTaggableOn::Tag.destroy_all(name: '')
+   ActsAsTaggableOn::Tag.destroy_all(name: %w(navarro etagwerker mattvvhat Assembly angels  capital combinations turbulenz semerda))
```

Comment:
Why are we removing all of them then? Shouldn't we delete all but one and update the corresponding `tag_id`s in the `taggings` table?